index.html

```diff
@@ -57,14 +57,16 @@ <h3>Rules</h3>
          </div>         
       </div>
       <div id="table-wrapper">
-         <div id="result-table">
-            <table>
+	  <table>
```

Comment:
Maintain indentation
```suggestion
	  <table>
            <thead>
               <tr>
                  <th>Guess</th>
                  <th>Result</th>
               </tr>
            </thead>
         </table>
         <div id="scroll-table">
            <table>
               <tbody id="result-body">
               </tbody>
            </table>
         </div>
```